projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.ts

```diff
@@ -53,6 +62,8 @@ export class DatagridNumericFilter<T = any>
 {
   @Input('clrFilterMinPlaceholder') minPlaceholder: string;
   @Input('clrFilterMaxPlaceholder') maxPlaceholder: string;
+  @Input('datagridNumericFilterFrom') datagridNumericFilterFrom: string;
+  @Input('datagridNumericFilterTo') datagridNumericFilterTo: string;
```

Comment:
```suggestion
  @Input('clrFilterMinValue') minLabel: string;
  @Input('clrFilterMaxValue') maxValue: string;
```

Name and order these consistently with `minPlaceholder` and `maxPlaceholder`.